app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: to_down_case(params[:session][:email])
+    if user&.authenticate(params[:session][:password])
+      log_in user
+      redirect_to user
+    else
+      flash.now[:danger] = t ".invalid"
+      render :new, status: :unprocessable_entity
```

Comment:
`status: :unprocessable_entity` có tác dụng gì em

Reply:
Em xin trả lời câu hỏi này như sau:
- Unprecessable Entity response status với mã 422 là thông báo rằng máy chủ nhận được và hiểu yêu cầu, nhưng không thể xử lý yêu cầu.
- Trong ngữ cảnh trên, em phải sử dụng nó vì turborails không cho phép render với status 200 (vì stateful post request), cho nên em đã dùng code này. Em tham khảo link này ạ : https://stackoverflow.com/questions/70606742/rendering-a-view-after-form-submission-not-working-correctly-in-rails-7